app/collins/models/Asset.scala

```diff
@@ -266,7 +266,7 @@ object Asset extends Schema with AnormAdapter[Asset] with AssetKeys {
     val findLocations = find(PageParams(0, 50, "ASC", "tag"), AttributeResolver.emptyResultTuple, instanceFinder)
       .items
       .collect { case a: Asset => a }
-      .filter { _.tag != MultiCollinsConfig.thisInstance }
+      .filter(asset => { !MultiCollinsConfig.thisInstance.contains(asset.tag) })
```

Comment:
t would be nice to add a comment explaining this.  On the surface it looks like you are removing any assets that exist in the current instance, instead of assets returned from the direct (vs multi-collins) query.  [unless I'm misinterpreting things...]